### PR TITLE
feat: ✨ add timestamps

### PR DESCRIPTION
**Pull request by jalezi**
Both timestamps (doctors and institutions) are available.
ATM using only doctors.

Check both languages.

What should we do with institutions ts?

### CODE REVIEW

File: src/components/Shared/FooterInfo.js [hunk 1]
```diff
@@ -52,10 +59,8 @@ const FooterInfoCard = function FooterInfoCard({ isDrPage = false }) {
       >
         GURS
       </a>
-      {/*
       <br />
-      {t('footer.lastChange')}: <strong>{lastChange()}</strong>
-      */}
+      {t('footer.lastChange')}: <strong>{lastChange(drTs)}</strong>
```
Comment (stefanb) [inline src/components/Shared/FooterInfo.js:63]: Could this be changed to use i18next interpolation for better flexibility in translations?
```suggestion
      {t('footer.lastChange', { date: new Date(drTs) })}
```

or remove the placeholders from translations: https://github.com/sledilnik/zdravniki/blob/947357458bafa53846c299c288b2d89173458ada/src/locales/en.json#L79

Comment (jalezi) [reply src/components/Shared/FooterInfo.js:63]: sure. I did't know it's possible.

**Comment by lukarenko** [comment]
Do not request institutions timestamp - not needed

File: src/context/timestampsContext.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+const TimestampsContext = createContext();
+
+export const TimestampsConsumer = TimestampsContext.Consumer;
+
+const TimestampsProvider = function TimestampsProvider({ children }) {
+  const [drTs, setDrTs] = useState(null);
+  const [instTs, setInstTs] = useState(null);
+
+  const drTsRequest = useFetchTs(CSV_URL.DOCTORS_TS);
+  const instTsRequest = useFetchTs(CSV_URL.INSTITUTIONS_TS);
```
Comment (lukarenko) [inline src/context/timestampsContext.js:15]: I would never request `institutions.csv.timestamp` - no need as we only need one timestamp

**Review by lukarenko — COMMENTED** [review]
Do not request institutions timestamp

File: src/context/timestampsContext.js (hunk 2 above)
Comment (jalezi) [reply src/context/timestampsContext.js]: done.

File: src/components/Shared/FooterInfo.js (hunk 1 above)
Comment (jalezi) [reply src/components/Shared/FooterInfo.js]: Done. Slightly different as you suggested.

**Comment by lukarenko** [comment]
@jalezi it seems that you have broken date formatting
image

**Comment by jalezi** [comment]
ups

**Comment by lukarenko** [comment]
@jalezi why do we not use i18next formatting, like on covid-19 project?
It would be also great to add time:
        "updated": "Osveženo: {{ date, %A, %e. %B %Y }}",
image

**Comment by jalezi** [comment]
@lukarenko I think @bananica tried, I have tried. Couldn't figured out. I will try again at some point. Didn't know you want time. It can be also relative like on social

relative social
.

**Comment by jalezi** [comment]
@lukarenko I guess I figured it out.

**Pull request merged**